### DISCUSSION TITLE
How do I use the new web browsing capabilities?

### DISCUSSION THREAD

**Discussion by mak448a** [open-webui/open-webui#2764] [Q&A] [answered]
In the release notes for https://github.com/open-webui/open-webui/releases/tag/v0.2.1, it says that it implemented search capabilities. Where can I find the setting to enable it? I tried looking in settings in the ui.

**Comment by josephbkfung** [comment, marked as answer]
The web searching toggle switch is hidden in the plus sign from the left side of the message field.  <img width="362" alt="image" src="https://github.com/open-webui/open-webui/assets/48842605/f28f3b89-6f9c-4c2e-a618-1c08069f91fa">

Of course you still need to configure the search api key following the documenetation https://docs.openwebui.com/getting-started/env-configuration#web-search

**Reply by mak448a** [reply]
Thanks!

**Reply by pbpippen** [reply]
I have the latest version v.0.2.4 and when I click the plus sign, I only see Upload Files. Is there anything I am doing wrong? I would like to enable Web Search.

![image](https://github.com/open-webui/open-webui/assets/74429444/66f48790-dc7d-4973-83ab-be5efb59cc58)

**Reply by pbpippen** [reply]
Oops, guess I forgot to set the environment variable ENABLE_RAG_WEB_SEARCH. I see it now

**Reply by Mythendros** [reply]
How did you set it?

**Reply by pbpippen** [reply]
In the docker start command. Or you can do it from UI (Workspace - Documents - Document Settings - Web Params)
